frontend_lib/src/component/Input/AutoCompleteContainer/AutoCompleteContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import classnames from 'classnames'
+import Avatar from '../../Avatar/Avatar'
+import PropTypes from 'prop-types'
+
+export const AutoCompleteContainer = props => {
+  return (
+    <div className='autocomplete' style={props.style}>
+      {props.autoCompleteItemList.map((m, i) => (
+        <>
+          <div
+            className={
+              classnames('autocomplete__item', { autocomplete__item__active: props.autoCompleteCursorPosition === i })
+            }
+            key={m.mention}
+            onClick={() => props.onClickAutoCompleteItem(m)}
+          >
+            {m.username && <Avatar width='15px' style={{ margin: '5px' }} publicName={m.detail} />}
+            <b className='autocomplete__item__mention'>@{m.mention}</b> - {m.detail}
```

Comment:
you are mixing a generic component name with a class that seems to indicate that your component can only handle mentions, there might be a bad naming somewhere